sources/draft-camelot-holy-grenade.adoc

```diff
@@ -358,58 +234,22 @@ recount here through monospace font and multiple spaces.
 [[killer_rabbit_caerbannog]]
 === The Killer Rabbit of Caerbannog
 
-////
-// end::xref[]
-[.comment]
-end::xref1[]
-////
-
-////
-[.comment]
-tag::relref1[]
-// tag::relref[]
-////
-
 The *Killer Rabbit of Caerbannog*, that most formidable foe of
 the Knights and of all that is holy or carrot-like, has been
 depicted diversely in lay and in song. We venture to say,
-_contra_ the claim made in <<RFC8140,of,section=4.1>>,
+_contra_ the claim made in <<RFC8140,section=4.1>> of Ze Vompyre,
```

Comment:
The "of," markup I have used is legal IETF-specific metanorma: https://www.metanorma.org/author/ietf/topics/markup/#citations

Reply:
Thank you, @opoudjis ! I did not use it before.